### PR TITLE
🌐[i18n-KO] Translate `autoclass_tutorial` to Korean and Fix the typo of `quicktour`

### PR DESCRIPTION
**Pull request by gabrielwithappy**
# What does this PR do?

Translated the `autoclass_tutorial.mdx` file of the documentation to Korean and fix the typo of `quicktour`
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Team PseudoLab, may you please review this PR?
@0525hhgus, @KIHOON71, @sim-so, @gabrielwithappy, @HanNayeoniee, @wonhyeongseo, @jungnerd

May you please review this PR?
@sgugger, @ArthurZucker, @eunseojo 

## Review result
- [x] fix a wrong source code link of functions in the document 
> Links for API documents are not activated. I checked other language documents have same problem. I think it will be fixed when API documents are translated.

### CODE REVIEW

**Review by sgugger — APPROVED** [review]
Thanks for the translation! Let us know when it's good to merge.

**Comment by gabrielwithappy** [comment]
need to add the Head #1 english link of `Load pretrained instances with an AutoClass`

**Review by gabrielwithappy — COMMENTED** [review]
checked

File: docs/source/ko/autoclass_tutorial.mdx [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+<Tip warning={true}>
+
+PyTorch모델의 경우 `from_pretrained()` 메서드는 내부적으로 피클을 사용하여 안전하지 않은 것으로 알려진 `torch.load()`를 사용합니다. 
+일반적으로 신뢰할 수 없는 소스에서 가져왔거나 변조되었을 수 있는 모델은 로드하지 마세요. 허깅 페이스 허브에서 호스팅되는 공개 모델의 경우 이러한 보안 위험이 부분적으로 완화되며, 각 커밋 시 멀웨어를 [검사합니다](https://huggingface.co/docs/hub/security-malware). GPG를 사용해 서명된 [커밋 검증](https://huggingface.co/docs/hub/security-gpg#signing-commits-with-gpg)과 같은 모범사례는 [문서](https://huggingface.co/docs/hub/security)를 참조하세요.
+
+텐서플로우와 플랙스 체크포인트는 영향을 받지 않으며, `from_pretrained`메서드에 `from_tf` 와 `from_flax` 키워드 가변인자를 사용하여 이 문제를 우회할 수 있습니다.
+
+</Tip>
+
+일반적으로 AutoTokenizer 클래스와 AutoModelFor 클래스를 사용하여 미리 학습된 모델 인스턴스를 로드하는 것이 좋습니다. 이렇게 하면 매번 올바른 아키텍처를 로드할 수 있습니다. 다음 [튜토리얼](preprocessing)에서는 새롭게 로드한 tokenizer, 이미지 프로세서, 특징 추출기를 사용하여 미세 조정용 데이터 세트를 전처리하는 방법에 대해 알아봅니다.
```
[the remark this answers is not in the document]
Comment (HanNayeoniee) [reply docs/source/ko/autoclass_tutorial.mdx]: `미세 튜닝`으로 아직 수정하지 않으신 것 같은데, 어떤 단어로 번역할지 결정되서 resolved 하신건가요?

File: docs/source/ko/quicktour.mdx [hunk 2]
```diff
@@ -168,7 +168,7 @@ label: NEGATIVE, with score: 0.5309
 
 ### AutoTokenizer
 
-토큰나이저는 전처리를 담당하며, 텍스트를 모델이 받을 숫자 배열로 바꿉니다. 토큰화 과정에는 단어를 어디에서 끊을지, 얼만큼 나눌지 등을 포함한 여러 규칙이 있습니다. 자세한 내용은 [토크나이저 요약](./tokenizer_summary)를 확인해주세요. 제일 중요한 점은 모델이 훈련됐을 때와 동일한 토큰화 규칙을 쓰도록 동일한 모델 이름으로 토크나이저 인스턴스를 만들어야 합니다.
+토크나이저는 전처리를 담당하며, 텍스트를 모델이 받을 숫자 배열로 바꿉니다. 토큰화 과정에는 단어를 어디에서 끊을지, 얼만큼 나눌지 등을 포함한 여러 규칙이 있습니다. 자세한 내용은 [토크나이저 요약](./tokenizer_summary)를 확인해주세요. 제일 중요한 점은 모델이 훈련됐을 때와 동일한 토큰화 규칙을 쓰도록 동일한 모델 이름으로 토크나이저 인스턴스를 만들어야 합니다.
```
Comment (wonhyeongseo) [inline docs/source/ko/quicktour.mdx:171]: Thank you for catching this sir!

File: docs/source/ko/autoclass_tutorial.mdx [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+* 사전 학습된 토크나이저 로드하기.
+* 사전 학습된 이미지 프로세서 로드하기
+* 사전 학습된 특징 추출기 로드하기.
+* 사전 훈련된 프로세서 로드하기.
+* 사전 학습된 모델 로드하기.
+
+## AutoTokenizer
+
+거의 모든 NLP 작업은 토크나이저로 시작됩니다. 토크나이저는 사용자의 입력을 모델에서 처리할 수 있는 형식으로 변환합니다.
+[`AutoTokenizer.from_pretrained`]로 토크나이저를 로드합니다:
```
Comment (HanNayeoniee) [inline docs/source/ko/autoclass_tutorial.mdx:34]: [`AutoTokenizer.from_pretrained`](https://huggingface.co/docs/transformers/v4.27.2/en/model_doc/auto#transformers.AutoTokenizer.from_pretrained)로 토크나이저를 로드합니다:

File: docs/source/ko/autoclass_tutorial.mdx [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+>>> image_processor = AutoImageProcessor.from_pretrained("google/vit-base-patch16-224")
+```
+
+
+## AutoFeatureExtractor
+
+오디오 작업의 경우 특징 추출기가 오디오 신호를 올바른 입력 형식으로 처리합니다.
+
+[`AutoFeatureExtractor.from_pretrained`]로 특징 추출기를 로드합니다:
```
Comment (HanNayeoniee) [inline docs/source/ko/autoclass_tutorial.mdx:67]: [`AutoFeatureExtractor.from_pretrained`](https://huggingface.co/docs/transformers/v4.27.2/en/model_doc/auto#transformers.AutoFeatureExtractor.from_pretrained)로 특징 추출기를 로드합니다:

File: docs/source/ko/autoclass_tutorial.mdx [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+>>> feature_extractor = AutoFeatureExtractor.from_pretrained(
+...     "ehcalabres/wav2vec2-lg-xlsr-en-speech-emotion-recognition"
+... )
+```
+
+## AutoProcessor
+
+멀티모달 작업에는 두 가지 유형의 전처리 도구를 결합한 프로세서가 필요합니다. 예를 들어 LayoutLMV2 모델에는 이미지를 처리하는 이미지 프로세서와 텍스트를 처리하는 토크나이저가 필요하며, 프로세서는 이 두 가지를 결합합니다.
+
+[`AutoProcessor.from_pretrained()`]로 프로세서를 로드합니다:
```
Comment (HanNayeoniee) [inline docs/source/ko/autoclass_tutorial.mdx:81]: [`AutoProcessor.from_pretrained()`](https://huggingface.co/docs/transformers/v4.27.2/en/model_doc/auto#transformers.AutoProcessor.from_pretrained)로 프로세서를 로드합니다:

File: docs/source/ko/autoclass_tutorial.mdx [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+>>> from transformers import AutoProcessor
+
+>>> processor = AutoProcessor.from_pretrained("microsoft/layoutlmv2-base-uncased")
+```
+
+## AutoModel
+
+<frameworkcontent>
+<pt>
+마지막으로 AutoModelFor클래스를 사용하면 주어진 작업에 대해 미리 학습된 모델을 로드할 수 있습니다 (사용 가능한 작업의 전체 목록은 [여기](model_doc/auto)를 참조하세요). 예를 들어, [`AutoModelForSequenceClassification.from_pretrained`]를 사용하여 시퀀스 분류용 모델을 로드할 수 있습니다:
```
Comment (HanNayeoniee) [inline docs/source/ko/autoclass_tutorial.mdx:93]: 마지막으로 AutoModelFor클래스를 사용하면 주어진 작업에 대해 미리 학습된 모델을 로드할 수 있습니다 (사용 가능한 작업의 전체 목록은 [여기](model_doc/auto)를 참조하세요). 예를 들어, [`AutoModelForSequenceClassification.from_pretrained`](https://huggingface.co/docs/transformers/v4.27.2/en/model_doc/auto#transformers.FlaxAutoModelForVision2Seq.from_pretrained)를 사용하여 시퀀스 분류용 모델을 로드할 수 있습니다:

File: docs/source/ko/autoclass_tutorial.mdx [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+일반적으로 신뢰할 수 없는 소스에서 가져왔거나 변조되었을 수 있는 모델은 로드하지 마세요. 허깅 페이스 허브에서 호스팅되는 공개 모델의 경우 이러한 보안 위험이 부분적으로 완화되며, 각 커밋 시 멀웨어를 [검사합니다](https://huggingface.co/docs/hub/security-malware). GPG를 사용해 서명된 [커밋 검증](https://huggingface.co/docs/hub/security-gpg#signing-commits-with-gpg)과 같은 모범사례는 [문서](https://huggingface.co/docs/hub/security)를 참조하세요.
+
+텐서플로우와 Flax 체크포인트는 영향을 받지 않으며, `from_pretrained`메서드에 `from_tf` 와 `from_flax` 키워드 가변 인자를 사용하여 이 문제를 우회할 수 있습니다.
+
+</Tip>
+
+일반적으로 AutoTokenizer 클래스와 AutoModelFor 클래스를 사용하여 미리 학습된 모델 인스턴스를 로드하는 것이 좋습니다. 이렇게 하면 매번 올바른 아키텍처를 로드할 수 있습니다. 다음 [튜토리얼](preprocessing)에서는 새롭게 로드한 토크나이저, 이미지 프로세서, 특징 추출기를 사용하여 미세 튜닝용 데이터 세트를 전처리하는 방법에 대해 알아봅니다.
+</pt>
+<tf>
+마지막으로 `TFAutoModelFor` 클래스를 사용하면 주어진 작업에 대해 사전 훈련된 모델을 로드할 수 있습니다. (사용 가능한 작업의 전체 목록은 [여기](model_doc/auto)를 참조하세요. 예를 들어, [`TFAutoModelForSequenceClassification.from_pretrained`]로 시퀀스 분류를 위한 모델을 로드합니다:
```
Comment (HanNayeoniee) [inline docs/source/ko/autoclass_tutorial.mdx:121]: 마지막으로 `TFAutoModelFor` 클래스를 사용하면 주어진 작업에 대해 사전 훈련된 모델을 로드할 수 있습니다. (사용 가능한 작업의 전체 목록은 [여기](model_doc/auto)를 참조하세요. 예를 들어, [`TFAutoModelForSequenceClassification.from_pretrained`](https://huggingface.co/docs/transformers/v4.27.2/en/model_doc/auto#transformers.FlaxAutoModelForVision2Seq.from_pretrained)로 시퀀스 분류를 위한 모델을 로드합니다:

**Comment by gabrielwithappy** [comment]
음.. 좀 헷갈리네요. 링크가 안되는 이유가 뭔지 좀 알아봐야 될 것 같습니다.
I will check why the hyperlink does not work.

**Comment by gabrielwithappy** [comment]
Thank you.
I updated review action items @HanNayeoniee

**Comment by gabrielwithappy** [comment]
May you please review this PR?
@sgugger, @ArthurZucker, @eunseojo

**Comment by sgugger** [comment]
Thanks for your contribution!

**Review by wonhyeongseo — COMMENTED** [review]
Congratulations on your first PR merge for HuggingFace, @gabrielwithappy ! 🎉 🎉 🎉 
I'm really excited to continue learning/working with you, and hope we can create something fun as well.

P.S. I forgot to post my reviews 😅 May I leave them in a sub-issue as we did with @jungnerd and #22549 ? Most issues are minor, but there is an outstanding issue on L13 we need to resolve.

File: docs/source/ko/_toctree.yml [hunk 8]
```diff
@@ -9,8 +9,8 @@
 - sections:
   - local: in_translation
     title: (번역중) Pipelines for inference
-  - local: in_translation
-    title: (번역중) Load pretrained instances with an AutoClass
+  - local: autoclass_tutorial
+    title: 자동 클래스로 사전 학습된 인스턴스 로드하기
```
Comment (HanNayeoniee) [inline docs/source/ko/_toctree.yml:13]: 목차에 있는 제목이 문서의 제목과 다릅니다.
동일하게 `AutoClass로 사전 학습된 인스턴스 로드`로 수정하는건 어떨까요?

**Pull request merged**